test/jdk/java/util/logging/FileHandlerAccessTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2000, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This seems like a new test - should it only have:

```
 * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
 ```

Reply:
Thanks, will update